<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="HomePage">
     <property name="styleSheet">
      <string notr="true">background-color: #424242;</string>
     </property>
     <widget class="QLabel" name="Logo">
      <property name="geometry">
       <rect>
        <x>85</x>
        <y>180</y>
        <width>330</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>48</pointsize>
        <italic>false</italic>
        <bold>true</bold>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 48pt &quot;Roboto&quot;;
color: #FFFFFF;</string>
      </property>
      <property name="text">
       <string>MOTIMATE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LetsGoButton">
      <property name="geometry">
       <rect>
        <x>195</x>
        <y>300</y>
        <width>150</width>
        <height>40</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #0b7560;
font: 700 25pt &quot;Roboto&quot;;
color: #333333;
border-width: 3px;</string>
      </property>
      <property name="text">
       <string>Let's Go!</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="OptionsPage">
     <property name="styleSheet">
      <string notr="true">background-color: #424242;</string>
     </property>
     <widget class="QPushButton" name="SetGoalButton">
      <property name="geometry">
       <rect>
        <x>145</x>
        <y>180</y>
        <width>280</width>
        <height>60</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #0b7560;
font: 700 40pt &quot;Roboto&quot;;
color: #333333;</string>
      </property>
      <property name="text">
       <string>Set Goal</string>
      </property>
     </widget>
     <widget class="QPushButton" name="TrackGoalButton">
      <property name="geometry">
       <rect>
        <x>145</x>
        <y>260</y>
        <width>280</width>
        <height>60</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #0b7560;
font: 700 40pt &quot;Roboto&quot;;
color: #333333;</string>
      </property>
      <property name="text">
       <string>Track Goal</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ReturnButton1">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>10</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/Images/ReturnArrow.jpg);
background-position: center;
border-radius: 250px;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>TrackGoalButton</zorder>
     <zorder>SetGoalButton</zorder>
     <zorder>ReturnButton1</zorder>
    </widget>
    <widget class="QWidget" name="SetGoalPage">
     <property name="styleSheet">
      <string notr="true">background-color: #424242;</string>
     </property>
     <widget class="QLabel" name="Goaltext">
      <property name="geometry">
       <rect>
        <x>145</x>
        <y>210</y>
        <width>70</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 18pt &quot;Arial&quot;;
color: #99ccff;</string>
      </property>
      <property name="text">
       <string>Goal:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="EnterGoalBox">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>210</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #F0F0F0;</string>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Deadlinetext">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>270</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 18pt &quot;Arial&quot;;
color: #99ccff;</string>
      </property>
      <property name="text">
       <string>Deadline:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="EnterDeadlineBox">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>270</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #F0F0F0;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ReturnButton2">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>10</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/Images/ReturnArrow.jpg);
background-position: center;
border-radius: 250px;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="SubmitButton">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>330</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #0b7560;
font: 700 25pt &quot;Roboto&quot;;
color: #333333;
border-width: 3px;</string>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="TrackGoalPage">
     <property name="styleSheet">
      <string notr="true">background-color: #424242;</string>
     </property>
     <widget class="QLabel" name="CurrentGoalstext">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>200</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>9</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
color: black;
font: 700 9pt &quot;Segoe UI&quot;;
</string>
      </property>
      <property name="text">
       <string>Current Goals:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ReturnButton3">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>10</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/Images/ReturnArrow.jpg);
background-position: center;
border-radius: 250px;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="DaysLeftText">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>200</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
color: black;
font: 700 9pt &quot;Segoe UI&quot;;
</string>
      </property>
      <property name="text">
       <string>Days Left:</string>
      </property>
     </widget>
     <widget class="QListWidget" name="GoalsList">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>230</y>
        <width>151</width>
        <height>131</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;</string>
      </property>
     </widget>
     <widget class="QListWidget" name="DeadlineList">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>230</y>
        <width>151</width>
        <height>131</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
